OSDOCS-5259: MicroShift sos reports (#57828)
openshift/openshift-docs: microshift_support/microshift-sos-report.adoc

:_content-type: ASSEMBLY
[id="microshift-sos-report"]
= MicroShift sos report
include::_attributes/attributes-microshift.adoc[]
:context: microshift-sos-report

toc::[]

[role="_abstract"]

> `sos` is a data collection tool composed of specialized plugins. An `sos report` will generate a detailed report with all the enabled plugins and data from the different components and applications in a system.

Reviewer: I think users will care more about what the tool is for than how it's built.

```suggestion
`sos` is a tool for collecting troubleshooting information about a host. An `sos report` will generate a detailed report with all the enabled plugins and data from the different components and applications in a system.
```

:FeatureName: MicroShift
include::snippets/microshift-tech-preview-snip.adoc[leveloffset=+1]

include::modules/microshift-about-sos-reports.adoc[leveloffset=+1]
include::modules/microshift-gathering-sos-report.adoc[leveloffset=+1]
include::modules/microshift-providing-sos-report-support.adoc[leveloffset=+1]

[role="_additional-resources"]
[id="additional-resources_microshift-sos-report"]
== Additional resources

> * link:https://access.redhat.com/solutions/2112[How to provide files to Red Hat Support (vmcore, rhev logcollector, sosreports, heap dumps, log files, etc.]

Reviewer: We could consider linking to https://access.redhat.com/solutions/3592 here, too, for more details on sos report.
